app/controllers/blazer/base_controller.rb

```diff
@@ -50,6 +50,20 @@ def process_vars(statement, data_source)
       end
     end
 
+    def parse_smart_variables(var, data_source)
+      query = data_source.smart_variables[var]
+      if query.is_a? Hash
+        smart_var = query.map { |k,v| [k, v] }
```

Comment:
I believe this needs a `reverse` to be in the format of `{value1: "Label 1", value2: "Label 2"}`.